code/game/objects/items/weapons/melee/axes.dm

```diff
@@ -447,26 +498,36 @@
 	minstr = 11
 	max_blade_int = 300
 	smeltresult = /obj/item/ingot/steel
+	max_integrity = INTEGRITY_STRONGEST
+	smeltresult = /obj/item/ingot/steel
+	melting_material = /datum/material/steel
+	melt_amount = 150
+	sellprice = 90
 
-/obj/item/weapon/greataxe/steel/doublehead
-	force = 15
-	force_wielded = 35
+//
+/obj/item/weapon/greataxe/steel/doublehead // Trades more damage for being worse to parry with and easier to dodge of.
+	slowdown = 1.5 //HEAVY
 	possible_item_intents = list(/datum/intent/axe/cut, /datum/intent/axe/chop, /datum/intent/spear/bash) //bash is for nonlethal takedowns, only targets limbs
-	gripped_intents = list(/datum/intent/axe/cut/battle/greataxe, /datum/intent/axe/chop/battle/greataxe,  /datum/intent/spear/bash)
+	gripped_intents = list(/datum/intent/axe/cut/battle/greataxe/doublehead, /datum/intent/axe/chop/battle/greataxe/doublehead,  /datum/intent/spear/bash)
 	name = "double-headed steel greataxe"
 	desc = "A steel great axe with a wicked double-bladed head. Perfect for cutting either men or trees into stumps.."
 	icon_state = "doublegreataxe"
 	icon = 'icons/roguetown/weapons/64.dmi'
+	max_blade_int = 400
 	minstr = 12
+	melt_amount = 180
+	wbalance = VERY_EASY_TO_DODGE
+	wdefense = MEDIOCRE_PARRY
```

Comment:
this in practice will be very weak; slowdown so you can't outrun anyone, incredibly easy to parry and dodge and gives you no parry chance.

Reply:
ye i agree it was too much, it was back to the base 1 slowdown that polearms have

Reply:
This the trade it does, while polearms are mostly good at parrying, their cut dmg is reduced and their only good dmg with 2 reach is the stab, while the greataxe benefit is being 2 reach in both CUT and CHOP with great damage on those as well.

MEDIOCRE_PARRY does not make it easier to  parry them, it just make harder for the user to parry WITH the weapon, besides that it will do a skill vs skill check and spd> vs spd check, since most weapons balance are HARD_TO_DODGE or below that won't matter(because the mult is 1 or below) on the parry check, only dagger, rapier or peaseant flail has the VERY_HARD_TO_DODGE